xrdsst/controllers/cert.py

```diff
@@ -114,9 +114,10 @@ def remote_import_certificates(ss_configuration, security_server):
                     cert_data = cert_file.read()
                     cert_file.close()
                     token_cert_api.import_certificate(body=cert_data)
+                    BaseController.log_info("Imported certificate '" + certfile + "'")
                 except ApiException as err:
                     if err.status == 409 and err.body.count("certificate_already_exists"):
-                        print("Certificate '" + certfile + "' already imported.")
+                        BaseController.log_info("Certificate '" + certfile + "' already imported.")
```

Comment:
certfile and cert_file could cause ambiguity, you could change certfile to cert_filename for example.

Reply:
-> cert_file_loc